frontend/src/app/components/Banner.js

```diff
@@ -0,0 +1,37 @@
+import React, { PropTypes } from 'react';
+import classnames from 'classnames';
+
+import Copter from './Copter';
+import LayoutWrapper from './LayoutWrapper';
+
+const renderCondensedBanner = ({ children, dataL10nId = null }) => {
+  return (
+    <div className="banner banner--condensed">
+      <LayoutWrapper flexModifier="row-between-reverse">
+        <h2 className="banner__title" data-l10n-id={dataL10nId}>{ children }</h2>
+        <Copter small="true" />
+      </LayoutWrapper>
+    </div>
```

Comment:
I think we could get rid of this special case. The other uses of `Banner` do not include the `LayoutWrapper`, `Copter`, or `h2` elements. If you remove those, the only difference is that the condensed banner uses the `banner--condensed` class instead of the `banner--expanded` class.

I think you could get rid of the special case using something like this (note that you don't need the dataL10nId any more, either):

```js
const Banner = ({ children, condensed = false, background = false }) => {
  return (
    <div className={classnames('banner', {
      'banner--condensed': condensed,
      'banner--expanded': !condensed,
      'banner--background': background
    }>
      { children }
    </div>
  );
};
```

Then you could replace that [one condensed instance](https://github.com/mozilla/testpilot/pull/2132/files#diff-d3b4b0683f2471eaec03fd641e6de552R69):

```js
// before
<Banner condensed={true} dataL10nId="experimentsListCondensedHeader">
         Pick your experiments!
</Banner>

// after
<Banner condensed={true}>
  <LayoutWrapper flexModifier="row-between-reverse">
    <h2 className="banner__title" data-l10n-id="experimentsListCondensedHeader">
        Pick your experiments!
    </h2>
    <Copter small="true" />
```